### DISCUSSION TITLE
How to properly import and use this?

### DISCUSSION THREAD

**Discussion by bluewave41** [libusb/hidapi#654] [Q&A]
Forgive me, I'm a total Mac and C++ noob but it's not at all clear how to use this.

I've followed the build steps for CMake.

```
# precondition: create a <build dir> somewhere on the filesystem (preferably outside of the HIDAPI source)
# this is the place where all intermediate/build files are going to be located
cd <build dir>
# configure the build
cmake <HIDAPI source dir>
# build it!
cmake --build .
# install library; by default installs into /usr/local/
cmake --build . --target install
# NOTE: you need to run install command as root, to be able to install into /usr/local/
```

I created my library in XCode and added

`#include <hidapi.h>`

Only to be met with

`'hidapi.h' file not found`

The file exists in my `usr/local/include/hidapi` so I don't understand what it's complaining about. Do I need to manually add a reference to this or something? How do I even start here?

**Comment by Youw** [comment]
Try checking: https://github.com/libusb/hidapi/blob/master/BUILD.cmake.md#using-hidapi-in-a-cmake-project
